config/main.py

```diff
@@ -1286,6 +1286,60 @@ def incremental(file_name):
     command = "acl-loader update incremental {}".format(file_name)
     run_command(command)
 
+#
+# 'drops' group ('config drops ...')
+#
+@config.group()
+def drops():
+    """Drop counter related configuration tasks"""
+    pass
+
+#
+# 'init' subcommand ('config drops init')
+#
+@drops.command()
+@click.argument("counter_name", type=str, required=True)
+@click.argument("counter_type", type=str, required=True)
+@click.argument("reasons",      type=str, required=True)
+@click.option("-a", "--alias", type=str, help="Alias for this counter")
+@click.option("-g", "--group", type=str, help="Group for this counter")
+@click.option("-d", "--desc",  type=str, help="Description for this counter")
+@click.option('-v', '--verbose', is_flag=True, help="Enable verbose output")
+def init(counter_name, alias, group, counter_type, desc, reasons, verbose):
+    """Initialize a new drop counter"""
+    command = "dropconfig -c install -n '{}' -t '{}' -r '{}'".format(counter_name, counter_type, reasons)
+    if alias: command += " -a '{}'".format(alias)
+    if group: command += " -g '{}'".format(group)
+    if desc:  command += " -d '{}'".format(desc)
+
+    run_command(command, display_cmd=verbose)
+
+@drops.command()
+@click.argument("counter_name", type=str, required=True)
+@click.option('-v', '--verbose', is_flag=True, help="Enable verbose output")
+def delete(counter_name, verbose):
+    """Delete an existing drop counter"""
+    command = "dropconfig -c uninstall -n {}".format(counter_name)
```

Comment:
>counter_name [](start = 53, length = 12)

Use `[https://docs.python.org/3/library/shlex.html#shlex.quote`](https://docs.python.org/3/library/shlex.html#shlex.quote`) to prevent input injection risk.

Reply:
That is a super cool tool, thank you for sharing! I'll go ahead and update this.

Reply:
It looks like `shlex.quote` was added in python 3, which we don't support yet.

Reply:
https://docs.python.org/2/library/shlex.html#shlex.shlex.quotes

Reply:
Is it possible to change this script to python3 syntax?

Reply:
I don't think so but maybe @jleveque can confirm. In either case, python2 is being deprecated soon so we do need some plan to move over to python3 in SONiC.

Reply:
Agree. Please created an issue or add TODO comment in code.

Reply:
Created #721